### PR TITLE
add foundation metrics

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: test/functional/managedclusterinfo_test.go [hunk 1]
```diff
@@ -26,7 +26,9 @@ import (
 )
 
 const (
-	clusterDeploymentResponse = `# HELP acm_managed_cluster_info Managed cluster information
+	clusterDeploymentResponse = `# HELP acm_managed_cluster_addon_status_condition Managed cluster add-on status condition
+# TYPE acm_managed_cluster_addon_status_condition gauge
+# HELP acm_managed_cluster_info Managed cluster information
```
Comment (haoqing0110) [inline test/functional/managedclusterinfo_test.go:31]: Does this metrics need functional test?

Comment (elgnay) [reply test/functional/managedclusterinfo_test.go]: Functional test cases added for the new metrics.

**Review by haoqing0110 — APPROVED** [review]
/approve

**Pull request merged**